include/NovelRT.h

```diff
@@ -234,4 +249,19 @@ namespace NovelRT::Windowing {
 #include "NovelRT/SceneGraph/QuadTreeNode.h"
 #include "NovelRT/SceneGraph/Scene.h"
 
+// Exception types
+#include "NovelRT/Exceptions/NotInitializedException.h"
+#include "NovelRT/Exceptions/InitialisationFailureException.h"
+#include "NovelRT/Exceptions/RuntimeNotFoundException.h"
+#include "NovelRT/Exceptions/FileNotFoundException.h"
+#include "NovelRT/Exceptions/NotSupportedException.h"
+#include "NovelRT/Exceptions/CharacterNotFoundException.h"
+#include "NovelRT/Exceptions/NullPointerException.h"
+#include "NovelRT/Exceptions/OutOfMemoryException.h"
+#include "NovelRT/Exceptions/FunctionNotFoundException.h"
+#include "NovelRT/Exceptions/InvalidOperationException.h"
+#include "NovelRT/Exceptions/IOException.h"
+#include "NovelRT/Exceptions/CompilationErrorException.h"
+#include "NovelRT/Exceptions/OpenGLLinkageFailure.h"
```

Comment:
Please move these includes so they're in alphabetical order both namespace wise and type wise.

Reply:
This still isn't resolved afaict.

Reply:
This has been resolved in [Reordered to be alphabetical](https://github.com/novelrt/NovelRT/pull/241/commits/67e521867560f5f94c0379396e4dccb9efa9b3ba)

Reply:
No, because they are still after the scenegraph namespace.

Reply:
moved them to in between NovelRT/Animation/. and NovelRT/Graphics/.